src/Directory.Build.targets

```diff
@@ -27,7 +27,7 @@
     <PackageLicenseUrl><!-- Don't set this field (deprecated) --></PackageLicenseUrl>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
-    <PackageTags>Azure;Bicep;Types;Az</PackageTags>
+    <PackageTags>Azure;Bicep;Types</PackageTags>
```

Comment:
I'm not sure the `Az` tag is very useful (even for the `bicep-types-az` packages): https://www.nuget.org/packages?q=Tags%3A%22Az%22

Reply:
Good point, may as well fix it up there too when this new package is incorporated.